Trunk/c_dungeon.py

```diff
@@ -1,16 +1,16 @@
 import c_room
-import itertools
+import c_cell
 import string
 import random
-from collections import Iterable
 
 
-class dungeon():
+class Dungeon:
 	rooms = []
+	cells = {}
```

Comment:
These are class variables right now, meaning they'd be shared across instances of Dungeons. To make them instance variables they need to be defined in __init__